test/test_files/log_files/dread/dread_dread_multiworld.rdvgame

```diff
@@ -477,7 +477,7 @@
                 "Cataris/Transport to Dairon/Elevator to Dairon - Transport to Cataris": "Dairon/Transport to Cataris",
                 "Cataris/Teleport to Dairon/Teleporter to Dairon - Teleport to Cataris": "Dairon/Teleport to Cataris",
                 "Dairon/Teleport to Artaria/Teleporter to Artaria - Teleport to Dairon": "Artaria/Teleport to Dairon",
-                "Dairon/East Transport to Ferenia/Elevator to Ferenia - East Transport to Darion": "Ferenia/East Transport to Darion",
+                "Dairon/East Transport to Ferenia/Elevator to Ferenia - East Transport to Dairon": "Ferenia/East Transport to Dairon",
```

Comment:
This pr is **not** allowed to touch any rdvgame.

The tests must pass with the rdvgames being untouched, as they represent all the rdvgames our users have.